jekyll/_docs/configuration.md

```diff
@@ -5,54 +5,41 @@ categories: [getting-started,reference]
-can likely tell you the best way to solve the problem.
-
-<h2 id="phases">File structure and content</h2>
-
-The `circle.yml` file is made up of six primary sections.
-Each section represents a _phase_ of running your tests:
-
-*   **machine**: adjusting the VM to your preferences and requirements
-*   **checkout**: checking out and cloning your git repo
-*   **dependencies**: setting up your project's language-specific dependencies
-*   **database**: preparing the databases for your tests
-*   **test**: running your tests
-*   **deployment**: deploying your code to your web servers
-
-The `circle.yml`
-file contains another **general** section for general build-related configurations
-that are not related to a specific phase, and an **experimental** section for early access
-previews of configuration changes under consideration.
-
-**Remember**: most projects won't need to specify anything for many of the phases.
-
-The sections contain lists of bash commands.  If you don't specify
-commands, CircleCI infers them from your code.  Commands are run in
-the order they appear in the file; all test commands are run to
-completion, but a non-zero exit code during the setup sections (`machine:, checkout:, dependencies:, database:`) will cause the
-build to fail early.  You can modify which&mdash;and
-when&mdash;commands are run by adding `override`,
-`pre` and/or `post` to adjust CircleCI's
-inferred commands.  Here's how it works:
-
-*   **pre**: commands run before CircleCI's inferred commands
-*   **override**: commands run instead of CircleCI's inferred commands
-*   **post**:  commands run after CircleCI's inferred commands
-
-Each command is run in a separate shell.
-As such, they do not share an environment with their predecessors, so be aware that
-`export foo=bar` in particular does not work.
-If you'd like to set an environment variable globally, you can specify them in the
-[Machine configuration](#machine) section, described below.
+CircleCI automatically infers settings from your code, so it's possible you won't need to add any custom configuration.
+
+If you _do_ need to tweak settings, you can create a `circle.yml` in your project's root directory. If this file exists, CircleCI will read it each time it runs a build.
+
+For a rough idea of what a `circle.yml` looks like, check out our [sample file]({{site.baseurl}}/config-sample/). Otherwise, read on for a more detailed look at each piece of a `circle.yml` file.
+
+<h2 id="phases">File Structure and Content</h2>
+
+The `circle.yml` file has six primary sections. Each section represents a _phase_ of the Build-Test-Deploy process:
+
+- **`machine`**: adjust the behavior of the virtual machine (VM)
+- **`checkout`**: checkout and clone code from a repository
+- **`dependencies`**: install your project's language-specific dependencies
+- **`database`**: prepare a database for tests
+- **`test`**: run your tests
+- **`deployment`**: deploy your code to your web servers
```

Comment:
We should include https://github.com/circleci/circleci-docs/pull/457 here as well. Perhaps merge that first and then we update here?